sqlite-android/build.gradle

```diff
@@ -13,7 +13,7 @@ description = 'Android SQLite compatibility library'
 android {
     compileSdkVersion 30
     buildToolsVersion "30.0.3"
-    ndkVersion '21.4.7075529'
+    ndkVersion '21.1.6352462'
```

Comment:
jitpack fails to find the ndk if you don't use the version they attempt, this is their version - from trial and error